docs/requirements.in

```diff
@@ -2,4 +2,4 @@ breathe
 myst-parser
 sphinx
 sphinx_rtd_theme
-docutils<0.17.0 # see https://github.com/readthedocs/sphinx_rtd_theme/issues/1115
+docutils
```

Comment:
Is the issue not relevant for us anymore or should we maybe force `>0.17.0` just to be safe?

Reply:
Yeah let me do that.